packages/eui/src/components/drag_and_drop/draggable.tsx

```diff
@@ -42,6 +43,13 @@ export interface EuiDraggableProps
    * Whether the item is currently in a position to be removed
    */
   isRemovable?: boolean;
+  /**
+   * Whether the currently dragged item is cloned into a portal in the body. This settings will
+   * ensure that drag & drop still works as expected within stacking contexts (e.g. within EuiFlyout and EuiModal).
+   * Make sure to apply styles directly to the Draggable content as relative styling from an outside
```

Comment:
[nit] Let's add an extra newline here to make the documentation a bit more readable
```suggestion
   *
   * Make sure to apply styles directly to the Draggable content as relative styling from an outside
```